OptimoveSDK/optimove-sdk/src/main/AndroidManifest.xml

```diff
@@ -22,4 +22,23 @@
             android:theme="@android:style/Theme.Translucent"
             android:exported="false" />
     </application>
+
+    <!-- https://developer.android.com/training/package-visibility/use-cases#check-browser-available -->
+    <queries>
+        <intent>
+            <action android:name="android.intent.action.VIEW" />
+            <category android:name="android.intent.category.BROWSABLE" />
+            <data android:scheme="https" />
+        </intent>
```

Comment:
can this be shortened to 

```
<intent>
            <action android:name="android.intent.action.VIEW" />
            <category android:name="android.intent.category.BROWSABLE" />
            <data android:scheme="https" />
            <data android:scheme="http" />
            <data android:scheme="market" />
        </intent>
```

Reply:
Seems like intent tags are are slightly different from intent-filter tags. Intent tag may have at most one data scheme

Reply:
oh, ok